extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -110,6 +110,7 @@ export class AmpSidebar extends AMP.BaseElement {
     const toolbarElements =
     Array.prototype.slice
       .call(this.element.querySelectorAll('nav[toolbar]'), 0);
+    toolbarElements.reverse();
```

Comment:
I am not re-ording the dom. I am reversing the elements within the array itself. I am doing this because then the toolbars would be added in reverse order. 

I can either do this, or traverse backward with a for(var i = toolbarElements.length) ... loop